.github/workflows/schema-check.yml

```diff
@@ -23,13 +23,14 @@ jobs:
         with:
           files: |
             **/*.ifc
-          files_separator: ";"
+          separator: "§"
       - name: Logging
         run: |
           echo 'The following files were identified for checking:' >> $GITHUB_STEP_SUMMARY
           echo '' >> $GITHUB_STEP_SUMMARY
-          IFS=';'; arrIN=(${{ steps.file_changes.outputs.all_changed_files }}); unset IFS;
-          for i in "${arrIN[@]}"; do
+          IFS='§'
```

Comment:
Why exactly do you need this 'IFS' set here?

Reply:
Resolved in call.